.github/workflows/frontend-ci.yml

```
@@ -0,0 +1,25 @@
+name: Frontend CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x, 16.x, 18.x]
```

(Apologies for the fly-by review, great PR ! )

```suggestion
        node-version: [20.x]
```
This suggestion is to target node 20 (https://github.com/XavB64/lowtrip/blob/998ac704b96d8d4089b720628f57e093db3b918a/frontend/package.json#L13) and thus divide by 3 the number of jobs run by only testing the one target version (#digitalsufficiency)